tools/docker/docker-compose.yaml

```diff
@@ -14,6 +14,8 @@ services:
     restart: always
     environment:
       - CL_DATABASE_URL
+#      - CL_SOLANA_CMD=chainlink-solana
+#      - CL_MEDIAN_CMD=chainlink-median
```

Comment:
disabling loop plugins?

Reply:
They are disabled by default. This is leftover after I tried to enable them - but IIRC this is rigged up to build a different Dockerfile that doesn't have them yet. I will remove this or add them to that file.

Reply:
- [ ] TODO

Reply:
No this has become stale, but I thought it might be a good reference for how things fit together in this sub dir.

Reply:
Sorry, is it clear what happens if you uncomment these lines? Is expected behavior to build a different docker image and enable these plugins? If yes, could you please test it and comment above.

Reply:
These lines are meant to enable LOOP Plugins. I expected it to "just work", because they are built in to the official docker image now (just disabled by default). However, something broken and I'm not sure what, because my explanation in the first comment seems wrong - the regular docker images are used :thinking: 

We should fix it, but it is orthogonal to running Solana, so it could be left out of this PR.